<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modify_user_info</class>
 <widget class="QWidget" name="modify_user_info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>361</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>BlankCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>modify_user_info</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#modify_user_info
{
	background-color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QLabel" name="lblPwd">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblHeight">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>身高：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSex">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>性别：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>133</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePwd">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>133</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="leHeight">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>230</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Yahei</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    font-family:Microsoft Yahei;
    color:white;
    background-color:rgb(14 , 150 , 254);
    border-radius:5px;
}

QPushButton:hover
{
    background-color:rgb(44 , 137 , 255);
}

QPushButton:pressed
{
    background-color:rgb(14 , 135 , 228);
    padding-left:3px;
    padding-top:3px;
}</string>
   </property>
   <property name="text">
    <string>确认  </string>
   </property>
   <property name="icon">
    <iconset resource="imageIcon.qrc">
     <normaloff>:/imgIcon/appbar.check.png</normaloff>:/imgIcon/appbar.check.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>76</width>
     <height>76</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    color:black;
    background-color:rgb(238 , 238 , 238);
    border: 1px solid rgb(183 , 183 , 183);
	border-radius:5px;
}

QPushButton:hover
{
    color:black;
    background-color:rgb(228 , 240 , 250);
    border: 1px solid rgb(15 , 150 , 255);
}

QPushButton:pressed
{
    color:black;
    background-color:rgb(204 , 228 , 247);
    border: 1px solid rgb(1 , 84 , 153);
    padding-left:3px;
    padding-top:3px;
}</string>
   </property>
   <property name="text">
    <string>取消  </string>
   </property>
   <property name="icon">
    <iconset resource="imageIcon.qrc">
     <normaloff>:/imgIcon/appbar.close.2.png</normaloff>:/imgIcon/appbar.close.2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>76</width>
     <height>76</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>cm</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPwdConfirm">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>160</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>再次输入密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leConfirmPwd">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>133</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>14</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    font-family:Microsoft Yahei;
    color:white;
	background-color:rgb(50 , 204 , 194);
    border-radius:5px;
}

QPushButton:hover
{
    background-color:rgb(50, 184, 179);
}

QPushButton:pressed
{
    background-color:rgb(41, 144, 163);
    padding-left:3px;
    padding-top:3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imageIcon.qrc">
     <normaloff>:/imgIcon/appbar.close.png</normaloff>:/imgIcon/appbar.close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>76</width>
     <height>76</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>-2</y>
     <width>76</width>
     <height>76</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/imgIcon/appbar.people.profile.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>15</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>修改用户个人信息</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rdoFemale">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string>女</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rdoMale">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string>男</string>
   </property>
  </widget>
  <zorder>lblPwd</zorder>
  <zorder>lblHeight</zorder>
  <zorder>lblSex</zorder>
  <zorder>leName</zorder>
  <zorder>lePwd</zorder>
  <zorder>leHeight</zorder>
  <zorder>btn_ok</zorder>
  <zorder>btn_cancel</zorder>
  <zorder>lblName</zorder>
  <zorder>label_5</zorder>
  <zorder>lblPwdConfirm</zorder>
  <zorder>leConfirmPwd</zorder>
  <zorder>btn_close</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>rdoFemale</zorder>
  <zorder>rdoMale</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imageIcon.qrc"/>
 </resources>
 <connections/>
</ui>
